lua/nvim-tree/actions/fs/remove-file.lua

```diff
@@ -44,10 +46,13 @@ local function clear_buffer(absolute_path)
   end
 end
 
+---@param cwd string
+---@return boolean|nil
 local function remove_dir(cwd)
   local handle = vim.loop.fs_scandir(cwd)
   if type(handle) == "string" then
-    return notify.error(handle)
+    notify.error(handle)
+    return
```

Comment:
In some cases I added an explicit nil return instead of returning the call of a function that returns nil.